rdf_differ/entrypoints/api/handlers.py

```diff
@@ -153,25 +153,33 @@ def delete_diff(dataset_id: str) -> tuple:
         raise NotFound(exception_text)  # 404
 
 
-def get_report(dataset_id: str) -> tuple:
+def get_report(dataset_id: str, application_profile: str = "diff_report") -> tuple:
     """
         Generate a dataset diff report
     :param dataset_id: The dataset identifier. This should be short alphanumeric string uniquely identifying the dataset
+    :param application_profile: The application profile identifier. This should be a text string
     :return: html report as attachment
     :rtype: file, int
     """
     logger.debug(f'start get report for {dataset_id} endpoint')
 
     dataset, _ = get_diff(dataset_id)  # potential 404
 
+    if application_profile not in config.RDF_DIFFER_APPLICATION_PROFILES_LIST:
+        exception_text = f"The chosen application profile does not exist. Existing application profiles:" \
+                         f" {','.join(config.RDF_DIFFER_APPLICATION_PROFILES_LIST)}"
+        logger.exception(exception_text)
+        raise UnprocessableEntity(exception_text)
+
     try:
         with tempfile.TemporaryDirectory() as temp_dir:
-            template_location = config.RDF_DIFFER_REPORT_TEMPLATE_LOCATION
+            template_location = config.RDF_DIFFER_REPORT_TEMPLATE_LOCATION + "/" + application_profile
```

Comment:
I think we should delegate this logic to the `config` module. 
```python
def get_aplication_profile_location(application_profile):
    return f'{RDF_DIFFER_REPORT_TEMPLATE_LOCATION}/{application_profile}'
```
and call it like:
```python
template_location = config.get_aplication_profile_location(application_profile)

Reply:
done